src/core/MOM_open_boundary.F90

```diff
@@ -528,13 +528,15 @@ subroutine open_boundary_config(G, US, param_file, OBC)
       OBC%add_tide_constituents = .false.
     endif
 
-    call get_param(param_file, mdl, "DEBUG", OBC%debug, default=.false.)
-    call get_param(param_file, mdl, "DEBUG_OBC", debug_OBC, default=.false.)
-    if (debug_OBC .or. OBC%debug) &
+    call get_param(param_file, mdl, "DEBUG", debug_OBC, default=.false.)
+    call get_param(param_file, mdl, "DEBUG_OBC", debug_OBC, default=debug_OBC)
+    if (debug_OBC) then
```

Comment:
With the newly revised logic, the `log_param()` call for DEBUG_OBC should probably be folded into the `get_param()` call for DEBUG_OBC, but with the added argument `do_not_log=.not.debug_OBC` so that it is only written the MOM_parameter_doc.debugging when DEBUG=True.  As written, DEBUG_OBC is only being logged when `DEBUG_OBC = .true.`, but we should also be logging it when `DEBUG=.true.` in order to advertise that this option exists and to replicate the previous logging behavior.

Reply:
Thanks, good point.